docs/http_api.rst

```diff
@@ -694,9 +706,15 @@ Teams
 .. _Delete team:
 .. http:delete:: /teams/(int:team_id)
 
-    Remove a team.
+    Archive a team. The team will no longer be shown when :ref:`listing teams
```

Comment:
Should we add `by default` here too, since we have the same `archived` parameter for teams?